### PR TITLE
remove koendv/FloatToString

### PR DESCRIPTION
**Pull request by koendv**
Intentionally remove https://github.com/koendv/FloatToString to avoid conflict with https://github.com/tedtoal/floatToString

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @koendv. I am confirming your request.

---

I really appreciate your care for fixing this semi-collision. We should probably adjust the submission checker system to detect this and block the submissions in order to ensure distinctive library names.

---

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/02/21 16:09:09 Loaded 6678 libraries from DB
Removing FloatToString
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/koendv/FloatToString not present
Success!
```

**Pull request merged**
